src/components/Sum.jsx

```diff
@@ -1 +1,21 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
the line is quite small so you could keep it simple
```suggestion
export const Sum = ({ a, b }) => (
```